pkg/services/accesscontrol/manager/manager_test.go

```diff
@@ -0,0 +1,117 @@
+package manager
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/grafana/grafana/pkg/api/routing"
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/registry"
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/grafana/grafana/pkg/services/accesscontrol/database"
+	actesting "github.com/grafana/grafana/pkg/services/accesscontrol/testing"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+func overrideAccessControlInRegistry(cfg *setting.Cfg) AccessControlService {
```

Comment:
I think these test utility functions should start with t.Helper().
```suggestion
func overrideAccessControlInRegistry(t *testing.TB, cfg *setting.Cfg) AccessControlService {
	t.Helper()

```